sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -91,6 +91,13 @@ private void canvasRestApiCall(HttpServletRequest request,
 		 }else {
 			url=canvasURL+pathInfo;
 		}
+		String sessionId = request.getSession().getId();
+		StringBuilder strgBlder =new StringBuilder();
+		strgBlder.append("Canvas API request URL  \"");
+		strgBlder.append(url);
+		strgBlder.append("\" Session Id: ");
+		strgBlder.append(sessionId);
+		M_log.info(strgBlder);
```

Comment:
This comment might require a separate jira. It would be good to log the amount of time that calls to outside services take so that we can tell what the outside services are doing even when running in production.